src/_P096_eInk.ino

```diff
@@ -0,0 +1,603 @@
+              eInkScreen->drawCircle(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), Plugin_096_ParseColor(sParams[3]));
+            }
+            else if(subcommand.equalsIgnoreCase(F("t")) && argCount == 7)
+            {
+              eInkScreen->drawTriangle(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), sParams[5].toInt(), Plugin_096_ParseColor(sParams[6]));
+            }           
+            else if(subcommand.equalsIgnoreCase(F("tf")) && argCount == 8)
+            {
+              eInkScreen->fillTriangle(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), sParams[5].toInt(), Plugin_096_ParseColor(sParams[7]));
+              eInkScreen->drawTriangle(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), sParams[5].toInt(), Plugin_096_ParseColor(sParams[6]));
+            }           
+            else if(subcommand.equalsIgnoreCase(F("rr")) && argCount == 6)
+            {
+              eInkScreen->drawRoundRect(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), Plugin_096_ParseColor(sParams[5]));
+            }          
+            else if(subcommand.equalsIgnoreCase(F("rrf")) && argCount == 7)
+            {
+              eInkScreen->fillRoundRect(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), Plugin_096_ParseColor(sParams[6]));
+              eInkScreen->drawRoundRect(sParams[0].toInt(), sParams[1].toInt(), sParams[2].toInt(), sParams[3].toInt(), sParams[4].toInt(), Plugin_096_ParseColor(sParams[5]));
+            } 
+            else if(subcommand.equalsIgnoreCase(F("px")) && argCount == 3)
+            {
+              eInkScreen->drawPixel(sParams[0].toInt(), sParams[1].toInt(), Plugin_096_ParseColor(sParams[2]));
+            } 
+            else 
+            {
+              success = false;
+            }
+          }
+                    
+        }
+        else
+        {
+          //invalid arguments
+          success = false;
+        }
+
+
+        if(!success)
+        {
+          if (loglevelActiveFor(LOG_LEVEL_INFO)) {
+            addLog(LOG_LEVEL_INFO, F("Fail to parse command correctly; please check API documentation"));
+            String log2  = F("Parsed command = \"");
+            log2 += string;
+            log2 += "\"";
+            addLog(LOG_LEVEL_INFO, log2);
+          }
+        }
+        else
+        {
+          if(plugin_096_sequence_in_progress == false)
+            eInkScreen->display();
+        }
+
+        String log;
+        log.reserve(110);                           // Prevent re-allocation
+        log = F("P096-eInk : WRITE = ");
+        log += tmpString;
+        SendStatus(event->Source, log);             // Reply (echo) to sender. This will print message on browser.  
```

Comment:
This is also printed when `!success`

Reply:
I just move this code in #ifdef DEBUG_LOG (to allow easy debugging in case of evolutions)